File: BBIS.Application/DTOs/UserAccount/UserProfileUpdateDto.cs
namespace BBIS.Application.DTOs.UserAccount
{
    public class UserProfileUpdateDto
    {
        public Guid UserId { get; set; }
        public string Username { get; set; }
        public string CurrentPassword { get; set; }
        public string NewPassword { get; set; }
    }
}

File: BBIS.Api/ClientApp/src/models/UserAccounts/UserProfileUpdateDto.ts
export interface IUserProfileUpdateDto {
    UserId: Guid;
    Username: string | null;
    CurrentPassword: string;
    NewPassword: string;
 }

export default class UserProfileUpdateDto implements IUserProfileUpdateDto{
    UserId: Guid = '';
    Username: string | null = '';
    CurrentPassword: string = '';
    NewPassword: string = '';
 }